src/main/java/org/sagebionetworks/web/client/DisplayUtils.java

```diff
@@ -773,12 +773,12 @@ public static Reference parseEntityVersionString(String entityVersion) {
 	 * @param bundle
 	 * @return
 	 */
-	public static PreviewFileHandle getPreviewFileHandle(EntityBundle bundle) {
-		PreviewFileHandle fileHandle = null;
+	public static CloudProviderFileHandleInterface getPreviewFileHandle(EntityBundle bundle) {
+		CloudProviderFileHandleInterface fileHandle = null;
 		if (bundle.getFileHandles() != null) {
 			for (FileHandle fh : bundle.getFileHandles()) {
-				if (fh instanceof PreviewFileHandle) {
-					fileHandle = (PreviewFileHandle) fh;
+				if (fh instanceof CloudProviderFileHandleInterface && ((CloudProviderFileHandleInterface) fh).getIsPreview()) {
```

Comment:
recommend small refactor to clean up and isolate logic:
public static boolean isPreviewFileHandle(FileHandle fh) {
return fh instanceof CloudProviderFileHandleInterface && ((CloudProviderFileHandleInterface) fh).getIsPreview()
}
and use this everywhere.
if (!isPreviewFileHandle(fh))...

Reply:
and updating the Synapse version to a real dev artifact (if these changes are in!) so that this can build.